src/toil/provisioners/gceProvisioner.py

```diff
@@ -0,0 +1,915 @@
+        #    raise ValueError("Cannot specify both 'image' and "
+        #                     "'ex_disks_gce_struct'.")
+
+        driver = self._getDriver()
+        if image and ex_image_family:
+            raise ValueError("Cannot specify both 'image' and "
+                             "'ex_image_family'")
+
+        location = location or driver.zone
+        if not hasattr(location, 'name'):
+            location = driver.ex_get_zone(location)
+        if not hasattr(size, 'name'):
+            size = driver.ex_get_size(size, location)
+        if not hasattr(ex_network, 'name'):
+            ex_network = driver.ex_get_network(ex_network)
+        if ex_subnetwork and not hasattr(ex_subnetwork, 'name'):
+            ex_subnetwork = \
+                driver.ex_get_subnetwork(ex_subnetwork,
+                                       region=driver._get_region_from_zone(
+                                           location))
+        if ex_image_family:
+            image = driver.ex_get_image_from_family(ex_image_family)
+        if image and not hasattr(image, 'name'):
+            image = driver.ex_get_image(image)
+        if not hasattr(ex_disk_type, 'name'):
+            ex_disk_type = driver.ex_get_disktype(ex_disk_type, zone=location)
+
+        node_attrs = {'size': size,
+                      'image': image,
+                      'location': location,
+                      'network': ex_network,
+                      'subnetwork': ex_subnetwork,
+                      'tags': ex_tags,
+                      'metadata': ex_metadata,
+                      'ignore_errors': ignore_errors,
+                      'use_existing_disk': use_existing_disk,
+                      'external_ip': external_ip,
+                      'ex_disk_type': ex_disk_type,
+                      'ex_disk_auto_delete': ex_disk_auto_delete,
+                      'ex_service_accounts': ex_service_accounts,
+                      'description': description,
+                      'ex_can_ip_forward': ex_can_ip_forward,
+                      'ex_disks_gce_struct': ex_disks_gce_struct,
+                      'ex_nic_gce_struct': ex_nic_gce_struct,
+                      'ex_on_host_maintenance': ex_on_host_maintenance,
+                      'ex_automatic_restart': ex_automatic_restart,
+                      'ex_preemptible': ex_preemptible}
+        # List for holding the status information for disk/node creation.
+        status_list = []
+
+        for i in range(number):
+            name = 'w' + bytes(uuid.uuid4()) #'%s-%03d' % (base_name, i)
+            status = {'name': name, 'node_response': None, 'node': None}
+            status_list.append(status)
+
+        start_time = time.time()
+        complete = False
+        while not complete:
+            if (time.time() - start_time >= timeout):
+                raise Exception("Timeout (%s sec) while waiting for multiple "
+                                "instances")
```

Comment:
`raise Exception("Timeout (%s sec) while waiting for multiple instances")`

Should be something like:

`raise Exception("Timeout (%s sec) while waiting for multiple instances" % str(timeout))`

Reply:
Monkey patch.